pkg/sources/github/github_test.go

```diff
@@ -94,27 +95,28 @@ func TestAddReposByOrg_IncludeRepos(t *testing.T) {
 	gock.New("https://api.github.com").
 		Get("/orgs/super-secret-org/repos").
 		Reply(200).
-		JSON([]map[string]string{
-			{"clone_url": "https://github.com/super-secret-repo.git", "full_name": "secret/super-secret-repo"},
-			{"clone_url": "https://github.com/super-secret-repo2.git", "full_name": "secret/super-secret-repo2"},
-			{"clone_url": "https://github.com/super-secret-repo2.git", "full_name": "secret/not-super-secret-repo"},
-		})
+		JSON(`[
+			{"full_name": "super-secret-org/super-secret-repo", "clone_url": "https://github.com/super-secret-org/super-secret-repo.git", "size": 1},
+			{"full_name": "super-secret-org/super-secret-repo2", "clone_url": "https://github.com/super-secret-org/super-secret-repo2.git", "size": 1},
+			{"full_name": "super-secret-org/not-super-secret-repo", "clone_url": "https://github.com/super-secret-org/not-super-secret-repo.git", "size": 1}
+		]`)
 
 	s := initTestSource(&sourcespb.GitHub{
 		Credential: &sourcespb.GitHub_Token{
 			Token: "super secret token",
 		},
-		Repositories:  []string{"secret/super*"},
+		IncludeRepos:  []string{"super-secret-org/super*"},
 		Organizations: []string{"super-secret-org"},
 	})
 	// gock works here because github.NewClient is using the default HTTP Transport
 	err := s.getReposByOrg(context.Background(), "super-secret-org")
 	assert.Nil(t, err)
 	assert.Equal(t, 2, s.filteredRepoCache.Count())
-	ok := s.filteredRepoCache.Exists("secret/super-secret-repo")
+	ok := s.filteredRepoCache.Exists("super-secret-org/super-secret-repo")
 	assert.True(t, ok)
-	ok = s.filteredRepoCache.Exists("secret/super-secret-repo2")
+	ok = s.filteredRepoCache.Exists("super-secret-org/super-secret-repo2")
 	assert.True(t, ok)
+	assert.False(t, gock.HasUnmatchedRequest())
```

Comment:
Add explicit failures if gock has unmatched requests. Otherwise, this will be logged but not fail the test which IMO isn't correct.

Reply:
Already paying off. It's already caught an (inconsequential) issue introduced by #2814.

(This gets called twice.)
https://github.com/rgmz/trufflehog/blob/5102e3ae1178e320e749ca14ac94b4073165c975/pkg/sources/github/github.go#L620-L622